src/pages/imprint.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { HelmetSeo } from '~/components/Helmet/HelmetSeo';
+import { LayoutArticle } from '~/components/Layout';
+import { ExternalLink } from '~/components/Links/ExternalLink';
+import BmdvFundingImage from '~/static/BMDV_Foerderung.svg';
```

Comment:
For SEO, we should try to avoid underscores and use "-" instead (and IMO also capital letters, but that more preference than SEO).

What would a good structure for the static folder be? I think we could start by mirroring the pages folders. So this file would be "static/page/imprint/bmdv-foerderung.svg".

In case it moves to the footer as well, we either have a duplicate in "components/footer/assets/bmdv-foerderung.svg" or find a shared assets structure or link it from the "wrong" folder as an exception.

Reply:
Mhm I would prefer to organize it in types of static content like `~/static/icons`, `~/static/photos`, `~/static/fonts`, and so on.

Reply:
@ohrie its really hard to find a thread if you resolve it. My email notification only shows the first send of a reply; to read or comment I have to search in the resolved threads.

---

On the topic: My goal would be to be able to remove a component or page and at the same time removing all their assets. This needs a structure based on content, not type.